Framework7/src/requests/validate.rb

```diff
@@ -1,14 +1,32 @@
 require_relative '../../src/data/comments'
 require 'json'
-class Comments_get
-  def validate_json(json_object)
-
+class Validate
+  def converting_to_json(json_object)
     @status_request, @json=json_object
-    hash_json = JSON.parse(@json)
-    hash_json.each do |index|
-      index.each do |key, value|
-      end
+    @hash_json = JSON.parse(@json)
+    return @status_request, @hash_json
+  end
+
+  def is_integer_value(integer_value)
+    integer_value.is_a? Integer
+  end
+
+  def is_string_value(string_value)
+    string_value.kind_of? String
+  end
+
+  def text_is_between_a_range(text, min_length_character, max_length_character)
+    if text.length >= min_length_character && text.length <= max_length_character
+      true
+    else
+      false
+    end
+  end
+
+  def is_a_date(date)
+    if date =~ (/^[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])T(0[0-9]|[1-5][0-9]):(0[0-9]|[1-5][0-9]):(0[0-9]|[1-5][0-9])Z$/)
```

Comment:
Maybe you can move the Regex into a Constant in base class in order that everybody can use it